sessions/js-modern-syntax/back-to-basics/index.html

```diff
@@ -3,17 +3,18 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="shortcut icon" href="./public/favicon.ico" type="image/x-icon" />
```

Comment:
```suggestion
    <link rel="icon" href="./public/favicon.ico" type="image/x-icon" />
```
`shortcut` shouldn't be used anymore (https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/rel#icon)

Reply:
Nice one, I didn't know about the `rel= shortcut `
The absolute path, when I try it (it maybe 2 year ago or something like that) not always worked..